src/assignmentNotification.test.js

```diff
@@ -87,15 +88,15 @@ describe('AssignmentNotification', () => {
       return testContext.notification.send().then(() => {
         expect(mockClient.history.post.length).toBe(2);
         expect(mockClient.history.post[0].data).toEqual(
-          JSON.stringify({
+          qs.stringify({
             visitor_id: 'visitorId',
             split_name: 'jabba',
             context: 'spec',
             mixpanel_result: undefined
```

Comment:
since the encoding matters, should we assert against an encoded string here (and maybe all the other places too)? i have no idea what `undefined` looks like after its encoded by `qs`.

But TIL:
```js
> JSON.stringify({blah: undefined})
"{}"
```